jax/lax/lax.py

```diff
@@ -1196,6 +1196,14 @@ def cumprod(operand: Array, axis: int) -> Array:
   """Computes a cumulative product along `axis`."""
   return cumprod_p.bind(operand, axis=int(axis))
 
+def cummax(operand: Array, axis: int) -> Array:
+  """Computes a cumulative maximum along `axis'."""
```

Comment:
Nit: here and below, `axis` should have back-ticks on both sides, for the sake of generated doc formatting (we use rst).

Reply:
Done.